packages/vscode-plugin/package.json

```diff
@@ -118,6 +118,12 @@
 					"default": false,
 					"description": "Skip linting link titles."
 				},
+				"harper.maxFileLength": {
```

Comment:
LGTM. The docs for the different editors also look good. Thanks for tackling this!